pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -393,6 +393,10 @@ func (c *awsElasticBlockStoreUnmounter) TearDown() error {
 
 // Unmounts the bind mount
 func (c *awsElasticBlockStoreUnmounter) TearDownAt(dir string) error {
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
```

Comment:
You will also need to modify the `UnmountPath(...)` method. The very last line of the common `UnmountPath(...)` should be returning an error instead of `nil`, e.g. `return fmt.Errorf("Failed to unmount volume dir... volume will not be deleted")`. However UnmountPath and these implementations assume that the are running on a directory that was mounted to a different device, not bind mounts (`IsLikelyNotMountPoint(...)` doesn't work for bind mounts). So that sucks. AFAIK, `TearDownAt(...)` for attachable volumes is called to tear down bind mounts (global mount to pod mount). So we can either modify the `UnmountPath(...)` method to take a parameter and fork it's behavior based on if it is unmounting a bind mount or a device mount. Or even better, we can replace `IsLikelyNotMountPoint(...)` once and for all with a real `IsMountPoint()` implementation that looks at the mount table instead of comparing devices (we can talk about this in person, because it is a bigger project, but one that we'd like to do for other reasons anyway).